.github/workflows/ci.yaml

```diff
@@ -46,9 +46,15 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-          bootstrap-options: "--agent-version 2.9.42"
+          bootstrap-options: "--model-default 'logging-config=<root>=INFO;unit=DEBUG' --agent-version 2.9.42 --constraints mem=1G"
       - name: Run integration bundle test
-        run: tox run -e integration -- --junit-xml=pytest_report.xml
+        run: tox run -e integration -- --junit-xml=pytest_report.xml --keep-models
+      - name: Setup upterm session
```

Comment:
should this be removed? also, `logging-config` above may be unnecessary after you are done debugging